libmscore/range.cpp

```diff
@@ -314,8 +314,7 @@ bool TrackList::write(Measure* measure) const
       Fraction pos;
       Measure* m       = measure;
       Score* score     = m->score();
-      Fraction rest    = m->len();
-      //Staff* staff     = score->staff(track2staff(_track));
+      Fraction remains = m->len();
```

Comment:
note that I changed this variable name to better represent that it represents remaining time in measure, but didn't change behavior around it.